### PR TITLE
Use sort by start/end to make traversing of who's oncall easier

### PR DESCRIPTION
**Pull request by kuba-kubula**
- introduces `who's oncall after`
- introduces `who's oncall before`
- uses sorted list of oncall shifts so it can be traversed for first (starts after now) or last (ends before now) shifts
- also allows for local running via `node ./node_modules/.bin/coffee index.coffee "who's oncall"`

### CODE REVIEW

File: src/scripts/pagerduty.coffee [hunk 1]
```diff
@@ -610,6 +610,14 @@ module.exports = (robot) ->
   robot.respond /who(?:’s|'s|s| is|se)? ((next (on call|oncall|on-call))|((on call|oncall|on-call) next))/i, (msg) ->
     getCustomOncalls 'next', msg
 
+  # who is on call after current shift?
+  robot.respond /who(?:’s|'s|s| is|se)? ((after (on call|oncall|on-call))|((on call|oncall|on-call) after))/i, (msg) ->
+    getCustomOncalls 'after', msg
+
+  # who is on call after current shift?
```
Comment (tomas-mechura) [inline src/scripts/pagerduty.coffee]: - nitpick: wrong comment
- wouldn't be better to use `who was on call before` ? as the past tense `was` is used already for the previous shift?

File: src/scripts/pagerduty-custom.coffee [hunk 2]
```diff
@@ -45,33 +101,54 @@ getCustomOncalls = (timeFrame, msg) ->
...
+    # plus added 3 layers of Incident-Command-Week-long
+    # Sum it up and you end-up at 66 "oncalls".
+    # We want to give it some space for overrides, 100 seems reasonable
+    query.limit = 100
+
   pagerduty.get('/oncalls', query, (err, json) ->
     if err
-      msg.send(err)
+      msg.send('Error:' + err)
+      return
```
Comment (tomas-mechura) [inline src/scripts/pagerduty-custom.coffee:123]: :+1:

**Review by tomas-mechura — COMMENTED** [review]
small nitpick, and question about past tense wen calling before.
otherwise LGTM.

File: src/scripts/pagerduty.coffee (hunk 1 above)
Comment (kuba-kubula) [reply src/scripts/pagerduty.coffee]: now allowed both, `'s` or `was` or whichever. And fixed comment

**Review by tomas-mechura — COMMENTED** [review]
LGTM

**Pull request merged**